yarn.lock

```diff
@@ -14807,7 +14807,7 @@ fork-ts-checker-webpack-plugin@^6.0.4:
     semver "^7.3.2"
     tapable "^1.0.0"
 
-form-data@^2.3.1, form-data@^2.3.3:
+form-data@^2.3.1:
```

Comment:
Do we care if we have an entry for 2.3.2 as well with a difference resolved version (2.3.3)?